kobuki_driver/src/driver/core_sensors.cpp

```diff
@@ -45,7 +45,7 @@ bool CoreSensors::serialise(ecl::PushAndPop<unsigned char> & byteStream)
 }
 bool CoreSensors::deserialise(ecl::PushAndPop<unsigned char> & byteStream)
 {
-  if (byteStream.size() < length+2)
+  if (byteStream.size() < (unsigned int)(length+2))
   {
```

Comment:
Should that be casted to size_t?

Reply:
I fixed it a different way by making the `2` a `2U`.